src/pages/Signup/Signup.js

```diff
@@ -1,8 +1,190 @@
-import React, { Component } from 'react';
+import React from 'react';
+import {
+  validationEmail,
+  validationPwd,
+  validationName,
+  validationPhone,
+} from './Validation';
+import './Signup.scss';
+
+class Signup extends React.Component {
+  nameRef = React.createRef();
+  phoneRef = React.createRef();
+  emailRef = React.createRef();
+  pwRef = React.createRef();
+
+  state = {
+    name: '',
+    email: '',
+    password: '',
+    phone: '',
+  };
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  checkAll = e => {
+    if (validationName(this.state.name) === false) {
+      alert('이름을 입력해주세요.');
+      this.nameRef.current.value = '';
+    }
+
+    if (validationPhone(this.state.phone) === false) {
+      alert('번호를 입력해주세요.');
+      this.phoneRef.current.value = '';
+    }
+
+    if (validationEmail(this.state.email) === false) {
+      alert('이메일 형식이 유효하지 않습니다.');
+      this.emailRef.current.value = '';
+    }
+
+    if (validationPwd(this.state.pw) === false) {
+      alert('영문,숫자,특수문자를 혼합하여 6~20자 이내로 입력해주세요.');
+      this.pwRef.current.value = '';
+    }
```

Comment:
boolean에 === true, === false 사용하시는 모습 자주보이는데 그럴필요 없습니다!
boolean은 이미 그 자체로 true or false 입니다!
```suggestion
    if (!validationPwd(this.state.pw)) {
      alert('영문,숫자,특수문자를 혼합하여 6~20자 이내로 입력해주세요.');
      this.pwRef.current.value = '';
    }
```

Reply:
수정 완료!!!